<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QFrame" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="SelectMenuFrame" name="selectMenuFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>540</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="AdvancedOptionFrame" name="advancedOptionFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>540</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="ConnectionMenuFrame" name="connectionMenuFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>540</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="CNX_LoadingBarWidget" name="loadingWidget" native="true">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>190</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedOptionFrame</class>
   <extends>QFrame</extends>
   <header>AdvancedOptionFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConnectionMenuFrame</class>
   <extends>QFrame</extends>
   <header>ConnectionMenuFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectMenuFrame</class>
   <extends>QFrame</extends>
   <header>SelectMenuFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CNX_LoadingBarWidget</class>
   <extends>QWidget</extends>
   <header>CNX_LoadingBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
